src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -1,12 +1,14 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import { useIntl } from 'react-intl';
 import HeaderSection from '../shared/HeaderSection/HeaderSection';
 import { InjectAppServices } from '../../services/pure-di';
 import { Breadcrumb, BreadcrumbItem } from '../shared/Breadcrumb/Breadcrumb';
 import { FormattedMessageMarkdown } from '../../i18n/FormattedMessageMarkdown';
+import { Switch } from '../shared/Switch/Switch';
 
 export const ContactPolicy = InjectAppServices(() => {
+  const [checked, setChecked] = useState(false);
```

Comment:
This state is refering to enabling/disabling contact policy. What do you think of adding a more descriptive state?
An example:
```suggestion
  const [active, toggleActive] = useState(false);
```